backend/app/views/spree/admin/customer_returns/new.html.erb

```diff
@@ -33,7 +33,7 @@
         </fieldset>
 
         <%= f.field_container :stock_location do %>
-          <%= f.label :stock_location, Spree.t(:stock_location) %>
+          <%= f.label Spree::StockLocation.model_name.human %>
```

Comment:
Word came from on high to completely ignore the select2 bug and not worry about it.  So just properly formed label and wait for them to patch it I guess.  Thanks for the help!